lease/lessor.go

```diff
@@ -484,23 +484,25 @@ func (le *lessor) initAndRecover() {
 }
 
 type Lease struct {
-	ID  LeaseID
-	TTL int64 // time to live in seconds
+	ID LeaseID
+
+	mu  sync.Mutex // to protect TTL
```

Comment:
This isn't necessary. If TTL is only set at creation time, there's no data race when fetching it thereafter.